File: Weatley.Backend/Weatley.Model/Entities/Room.cs
using System;
using System.Collections.Generic;
using System.Text;
using Weatley.Model.Enums;

namespace Weatley.Model.Entities
{
    public class Room : IEntityBase
    {
        public Guid Id { get; set; }
        public int RoomNumber { get; set; }
        public int FloorNumber { get; set; }
        public string RoomType { get; set; }
        public Hotel Hotel { get; set; }

        //Relation with Booking
        public ICollection<BookedRoom> BookedRooms { get; set; }
    }
}

File: Weatley.App/app/core/entities/room.ts
import { Hotel } from './hotel';

export class Room {
	id: string;
	roomNumber: number;
	floorNumber: number;
	roomType: string;
	price: number;
	hotel: Hotel;

	constructor(values: Object = {}) {
		Object.assign(this, values);
	}
}
